playbooks/robusta_playbooks/bash_enrichments.py

```diff
@@ -35,7 +44,33 @@ def node_bash_enricher(event: NodeEvent, params: BashParams):
 
     block_list: List[BaseBlock] = []
     exec_result = RobustaPod.exec_in_debugger_pod(
-        "node-bash-pod", node.metadata.name, params.bash_command, custom_annotations=params.custom_annotations
+        "node-bash-pod",
+        node.metadata.name,
+        params.bash_command,
+        custom_annotations=params.custom_annotations,
+        custom_volume_mounts=params.custom_volume_mounts,
+        custom_volumes=params.custom_volumes,
+    )
+    block_list.append(MarkdownBlock(f"Command results for *{params.bash_command}:*"))
+    block_list.append(MarkdownBlock(exec_result))
+    event.add_enrichment(block_list)
+
+
+@action
+def bash_enricher(event: ExecutionBaseEvent, params: BashParams):
+    """
+    Execute the specified bash command in a new bash pod instead of **pod_bash_enricher**  which runs on a target pod
+    Enrich the finding with the command results.
+    """
+
+    block_list: List[BaseBlock] = []
+    exec_result = RobustaPod.exec_in_debugger_pod(
+        "bash-pod",
+        None,
```

Comment:
Please fix node_ name param definition on exec_in_debugger_pod to include None as valid option.